extensions/amp-analytics/0.1/vendors.js

```diff
@@ -458,6 +459,15 @@ export const ANALYTICS_CONFIG = {
     },
   },
 
+  'adobeanalytics_nativeConfig': {
```

Comment:
Can you add an example in analytics.amp.html and some documentation to amp-analytics.md?